src/components/Tabs/Tabs.mdx

```diff
@@ -120,32 +130,26 @@ const animalCrossingData = [
 
 ## Responsive Mobile Carousel
 
-For both, the `tabsData` prop approach and the children component approach
-(described below in this document), the mobile carousel feature is built. This
-adds "previous" and "next" arrows in the tab list for users to scroll through
-all the tabs while the panel stays static.
-
-Note that for the children component approach, we expect a `TabList` component
-to be passed or else the mobile carousel will _not_ be built. More on this in
-the <Link href="#children-components" target="_self">Children Components</Link> section.
+- For mobile viewports, the `Tabs` component will generate a carousel feature.
+- The carousel feature adds `"previous"` and `"next"` buttons that can be used to scroll through
+  all tabs while the panel stays static.
 
 To view and test this in Storybook, go to the "Canvas" tab and change the
 viewport in the Storybook toolbar at the top of the page.
 
-## Scrollable Desktop Tab Set
-
-For desktop viewports (600px and greater), the full tab set scrolls left/right when
-the width of the tab set is greater than the width of the viewport.
+## Horizontal Scrolling
 
-To scroll horizontally, use the `shift + scroll wheel` combination.
+- When the width of the full tab set is greater than the available horizontal
+  space, the overflow will be hidden and horizontal scrolling will be enabled.
+- This functionality is the same for desktop, tablet, and mobile.
+- Horizontal scrolling can be accomplished by using the trackpad, the `shift + scroll wheel` combination, or the keyboard to tab through the available tabs.
 
 <Canvas of={TabsStories.ExtendedTabSetExample} />
 
 ## Callback Event Function
 
-If you need to keep track of the index of current tab that was selected, pass an
-`onChange` event handler to the `onChange` prop. This will return the index
-(0-based) of the tab selected.
+- The index (0-based) of the current tab will be returned to an event handler
```

Comment:
Is there a reason why we're using bullet points over text?

Reply:
The bullets make for easier consumption and help highlight what is important. My aim with using bullets has also been to simplify the text and to be less wordy.